src/main/java/com/xloop/resourceloop/createJob/Service/JobService.java

```diff
@@ -25,4 +26,25 @@ public Job save(Job job) {
         return jobRepository.save(job);
     }
 
+    public Job update(Job job){
```

Comment:
@WAQAR2211011KHICND is there any way to make this a batch operation? It's always better to make as few database operations as possible

Reply:
I think one option would be,
 - if I pass two object of job, one is previous job data another one is updated job data. and on spingboot we could analyze the json data and perform the updation on databse according to the changes.
 e.g.,
{
previousJobData:{jobObject},
currentJobData:{recentJobObject}
}

Reply:
is there any other option, for that?